packages/Webkul/Product/src/Type/Configurable.php

```diff
@@ -96,7 +96,7 @@ public function update(array $data, $id, $attribute = "id")
     {
         $product = parent::update($data, $id, $attribute);
 
-        if (request()->route()->getName() != 'admin.catalog.products.massupdate') {
+        if (app()->runningInConsole() || request()->route()->getName() != 'admin.catalog.products.massupdate') {
```

Comment:
There is no need for this check, just try with this,

~~~php
$route = request()->route() ? request()->route()->getName() : '';
~~~

Take a reference from `AbstractType` class.